lightstep/resource_metric_condition.go

```diff
@@ -554,12 +554,13 @@ func buildQueries(queriesIn []interface{}, includeSpansQuery bool) ([]client.Met
 				if err != nil {
 					return nil, err
 				}
+				display := query["display"].(string)
 				newQuery := client.MetricQueryWithAttributes{
 					Name:       query["query_name"].(string),
 					Type:       "spans_single",
 					Hidden:     query["hidden"].(bool),
-					Display:    query["display"].(string),
-					SpansQuery: buildSpansQuery(spansQuery),
+					Display:    display,
+					SpansQuery: buildSpansQuery(spansQuery, display),
```

Comment:
do we need to do this for metrics queries too?

Reply:
The only place these latency values are inserted are in this file.

Reply:
Specifically, a `Find in Files` for the value `99.9` returns only one hit.

Reply:
👍 probably worth testing a metrics heatmap, but i see no reason it shouldnt work